python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -329,6 +333,12 @@ def on_shutdown(self, worker_group: WorkerGroup,
                     backend_config: BackendConfig):
         raise NotImplementedError
 
+    def convert_checkpoint(self, checkpoint: Dict) -> Any:
+        pass
+
+    def unconvert_checkpoint(self, data: Any, to_gpu: bool = False) -> Dict:
+        pass
+
```

Comment:
Nit: I think the naming for these methods is quite confusing (maybe "convert" is a bit generic and does not immediately convey that this is for network transfer). What do you think about `serialize`/`deserialize` or `encode`/`decode`?

Reply:
+1